hubspot/search.go

```diff
@@ -35,11 +35,24 @@ func (c *Connector) Search(ctx context.Context, config SearchParams) (*common.Re
 }
 
 // BuildLastModifiedFilterGroup filters records modified since the given time.
-func BuildLastModifiedFilterGroup(since time.Time) Filter {
+// If the time is zero, it returns an empty filter. For contacts, it uses the
+// lastmodifieddate field. For other objects, it uses the hs_lastmodifieddate.
+// Read more: https://community.hubspot.com/t5/APIs-Integrations/CRM-V3-API-Search-issue-with-Contacts-when-using-Filters/m-p/324617
```

Comment:
Please add //nolint:lll to fix linter error

Reply:
Local & github action linters both aren't complaining about this, where did you see this? Merging it for now, will follow up with this if there's an error anywhere